gamestonk_terminal/portfolio/portfolio_controller.py

```diff
@@ -18,25 +17,38 @@
     check_positive_float,
     check_positive,
     EXPORT_ONLY_FIGURES_ALLOWED,
+    print_rich_table,
 )
 from gamestonk_terminal.menu import session
 
 from gamestonk_terminal.portfolio.portfolio_optimization import po_controller
 from gamestonk_terminal.portfolio import (
     portfolio_view,
     portfolio_model,
-    yfinance_model,
-    portfolio_helper,
 )
 from gamestonk_terminal.helper_funcs import parse_known_args_and_warn
 
 # pylint: disable=R1710,E1101,C0415
 
+portfolios_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), "portfolios")
+
 
 class PortfolioController(BaseController):
     """Portfolio Controller class"""
 
-    CHOICES_COMMANDS = ["load", "save", "show", "add", "rmv", "ar", "rmr", "al", "mdd"]
+    CHOICES_COMMANDS = [
+        "load",
+        "save",
+        "init",
+        "show",
+        "add",
+        "build",
+        "ar",
+        "rmr",
+        "al",
```

Comment:
The allocation (al) command got removed in this pr so it needs to be removed here. Is there a command that replaces this one? I've tested the show command, but it just shows the transactions and not the entire portfolio.

Reply:
I did remove it.  I can actually add it back this way and then you can add the improvements you had in mind: 
<img width="799" alt="Screen Shot 2022-02-02 at 11 45 20 AM" src="https://user-images.githubusercontent.com/18151143/152198445-011985df-ee38-48f8-bd49-be4b56f47b73.png">

Reply:
Sounds good! I'll add my improvements in another pr with other commands (VaR for portfolio and more risk metrics), but I also really like that you added this nice looking graph with the allocation over time. Great work!!!